Automattic/wc-calypso-bridge
# Add 'host' parameter to WooCommerce analytics collected by Jetpack

### Changes proposed in this Pull Request:



Closes #1241 .
- #1241






### How to test the changes in this Pull Request:





1. Install and activate https://jetpack.com/download-jetpack-beta/
2. Go to `wp-admin/admin.php?page=jetpack-beta&plugin=jetpack` and activate `Bleeding Edge` - This includes the merged PR Automattic/jetpack/pull/32052
3. Go to `wp-admin/admin.php?page=jetpack_modules` and make sure `WooCommerce Analytics` is enabled
4. Visit a product on your site, view source, and search for `woocommerceanalytics_product_view`
5. You should see `'host': 'ecommplan', `
6. Alternatively, use https://github.com/Automattic/tracks-vigilante-2-chrome-extension to see the events 

![image](https://github.com/Automattic/wc-calypso-bridge/assets/2484390/1347a47a-d43e-4a6f-8c3e-832bc8c6dc4b)




### Other information:




### FOR PR REVIEWER ONLY:
